### PR TITLE
Adds an image to the Morrow Command

### PR DESCRIPTION
**Pull request by morrowwolf**
https://i.imgur.com/67RIAY8.png

### CODE REVIEW

File: models/Discord/Commands/DiscordCommandMorrow.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ class DiscordCommandMorrow extends DiscordCommand {
 		"http://i0.kym-cdn.com/photos/images/original/000/912/556/ef5.png",
 		"https://i.imgur.com/LySOaCy.png",
 		"https://i.imgur.com/I6sdjhE.png"
```
Comment (nichlas0010) [inline models/Discord/Commands/DiscordCommandMorrow.js]: add a comma at the end of this

Comment (morrowwolf) [reply models/Discord/Commands/DiscordCommandMorrow.js]: errrr why?

Comment (nichlas0010) [reply models/Discord/Commands/DiscordCommandMorrow.js]: it's a list, all other lists have commas for all but the last entry.

https://github.com/yogstation13/yogbot/blob/e16db955597356bd35b1c31bb8768f07451a1912/models/Discord/Commands/DiscordCommandCodey.js#L15

https://github.com/yogstation13/yogbot/blob/e16db955597356bd35b1c31bb8768f07451a1912/models/Discord/Commands/DiscordCommandFennecs.js#L23

https://github.com/yogstation13/yogbot/blob/e16db955597356bd35b1c31bb8768f07451a1912/models/Discord/Commands/DiscordCommandKMC.js#L14

and so on.

Comment (morrowwolf) [reply models/Discord/Commands/DiscordCommandMorrow.js]: Oh shit I thought that was the last entry.  My bad will add comma.

**Pull request merged**